packages/ejson/has_size.js

```diff
@@ -0,0 +1,29 @@
+// Like `_.size(obj) === n` but faster by looking at at most `n+1`
+// items
+hasSize = function _hasSize(obj, n) {
```

Comment:
Looks like we might:

```js
X = (function () {                                                   
  function X() {}                                              
                                                               
  return X;                                              
})(); 
```

That's the compiled output for `X = function X() {}`.